examples/job-webhook-enhancer/handler.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/config"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+)
+
+type databricksWebhookContent struct {
+	EventType   string `json:"event_type"`
+	WorkspaceID int64  `json:"workspace_id"`
+	Run         struct {
+		RunID int64 `json:"run_id"`
+	}
+	Job struct {
+		JobID   int64  `json:"job_id"`
+		JobName string `json:"name"`
+	}
+}
+
+func databricksWebhookHandler(w http.ResponseWriter, r *http.Request) {
+	var jobMetadata jobs.Run
+	var err error = nil
+	JobWebhook := webhookParser(r)
+	switch EventType := JobWebhook.EventType; EventType {
+	case "jobs.on_start":
+		log.Printf("workspace_id: %v | job_name: %s | job_id: %v | run_id: %v | event: %s | current_action: None", JobWebhook.WorkspaceID, JobWebhook.Job.JobName, JobWebhook.Job.JobID, JobWebhook.Run.RunID, JobWebhook.EventType)
+		jobMetadata, err = databricksGetJobRunMetadata(JobWebhook)
+	case "jobs.on_failure":
+		log.Printf("workspace_id: %v | job_name: %s | job_id: %v | run_id: %v | event: %s | current_action: enrich_and_forward", JobWebhook.WorkspaceID, JobWebhook.Job.JobName, JobWebhook.Job.JobID, JobWebhook.Run.RunID, JobWebhook.EventType)
+		jobMetadata, err = databricksGetJobRunMetadata(JobWebhook)
+	case "jobs.on_success":
+		log.Printf("workspace_id: %v | job_name: %s | job_id: %v | run_id: %v | event: %s | current_action: None", JobWebhook.WorkspaceID, JobWebhook.Job.JobName, JobWebhook.Job.JobID, JobWebhook.Run.RunID, JobWebhook.EventType)
+		jobMetadata, err = databricksGetJobRunMetadata(JobWebhook)
+	default:
+		log.Printf("Unexpected event")
+	}
+	if err != nil {
+		panic(err)
+	}
+	SendDetailedWebhook(jobMetadata)
+}
+
+func SendDetailedWebhook(jobMetadata jobs.Run) {
```

Comment:
```suggestion
func sendDetailedWebhook(jobMetadata jobs.Run) {
```

un-export this function. there's no need to make this function public if we have just the `main` package.